### ISSUE TITLE
英文翻译/English TL + Contact

### ISSUE THREAD

**Issue by UnknownPi3** [HelloworldDk/dkmaiproj#2]
你好，我有兴趣自己构建这个项目，并且想传播它，看看我是否可以记录这个项目的英文版本，因为我们在美国没有很多 maimai 机器。 这条消息是用谷歌翻译的，但我会说普通话，虽然我在美国长大，但我的阅读和写作恐怕很差。 有联系方式吗？  Discord、微信等

Discord: pi3
WeChat: +1 9293295545
WeChat QR: https://imgur.com/a/86RCyfY
Telegram +1 9293295545

RAW: Hello, I’m interested in building this project myself and would like to spread it and see if I can document an English version of this project since we don’t have many maimai machines in the US. This message is translated using Google, but I can speak mandarin hereditarily, though I’m afraid my reading and writing is rather poor as grew up in the US. Do you have a contact method? Discord, WeChat, etc

**Comment by HelloworldDk** [comment]
I'm glad that my project can help more people.The telegram denied +86(Chinese) phone number to make private chat now so I sent a discord friend request to you.The new version of this guide is now under writing.Hearing that you want to help translating this guide,I felt very happy.We can translate the new ver together after finishing Chinese part.Begining translating now is not so good because during writing there will be many changes that will waste many translated parts.So just wait until the new ver's finish and then translate the completed ver will be better.Anyway thank you!

**Comment by UnknownPi3** [comment]
Thanks for replying! I'm not sure which is easier, but this first reply will be in English, let me know which you prefer.

I didn't even realize that a new version was being written, but I'm very happy that it is. It will be a bit before I can begin myself as I need to source parts, especially a display panel (Did you use a TV? Or was it just a LCD Panel, it may be cheaper for me to buy a used 43' TV and rip out the panel than anything else). AliExpress takes about a month's time to ship, but buying electronics in the US can also be much more expensive or just not available.

I'll most likely be confirming back and forth with you in Discord to clear up exactly what one would need to buy for this.
Thanks for your work on this project, I'll try my best to create a reproduction.

**Comment by HelloworldDk** [comment]
sorry but please wait until my summer vacation ,thanks
Now i am busy preparing for the final exam（期末考试） :(

**Comment by UnknownPi3** [comment]
与项目本身无关，但有些人是如何得到他们的纽扣和圆环的？我们在美国很难买到这些部件，例如，一个纽扣是22美元，约合158元人民币，更不用说我需要8个这样的部件，还必须3D打印连接件。您知道从哪里可以买到这些产品吗？它们不是国内（美国）生产的，我想使用或制作接近原装的部件。

Not related to the project itself, but how did some people get their buttons and rings? These parts are hard for us to come by in the U.S., for example, a single button is $22 USD or about 158 RMB, not to mention I would need 8 of these and have to 3D print the connector pieces. Do you have a source for where to get these? They're not produced domestically (United States) and I'd like to use or make as close as genuine parts

**Comment by HelloworldDk** [comment]
> 与项目本身无关，但有些人如何获得他们的按钮和戒指？我们在美国购买这些零件是非常困难的，比如一个按钮22美元，约合人民币158元，更何况我需要8个这样的零件，还得3D打印连接零件。你知道我在哪里可以买到这些产品吗？它们不是国产（美国）制造的，我想使用或制造接近原装零件。
> 
> Not related to the project itself, but how did some people get their buttons and rings? These parts are hard for us to come by in the U.S., for example, a single button is $22 USD or about 158 RMB, not to mention I would need 8 of these and have to 3D print the connector pieces. Do you have a source for where to get these? They're not produced domestically (United States) and I'd like to use or make as close as genuine parts

Yes the buttons and rings are expensive.In China,the 3rdparty copies of button are 80-90 CNY,and the rings are 60-80CNY.
3D print the rings is OK,there are someone selling 3d printed rings.I don't have the 3d model file of the rings but the rings don't have complicated structure so it looks easy to model it youself.
About buttons,one of my friends has made them using 3d print but with a non-offical structure.It means he designed a special structure to make up for the shortcomings of 3D printing(this sentense comes from google translate).
He hasn't shared the 3d model yet but I think you can take a look of his design https://www.bilibili.com/video/BV1YP41117Ht/

**Comment by UnknownPi3** [comment]
看起来很有趣，我今天能够前往当地的一家商场进行一些测量，如果没有参考，轮廓似乎很难完成。第一台机器将使用购买的按钮和 3D 打印的环。制作按钮原型可以等到机器启动后我可以自由测试。

你在哪里购买按钮？如果我必须从海外运送它们也没关系。我在这两个地方买的每个按钮的价格大约是160-180元，运费大约是360元，这是相当荒谬的。

Looks interesting, I was able to head into a local arcade to get some measurements today, the contours seem rather difficult to do without a reference. The first machine will use purchased buttons and a 3D-printed ring then. Prototyping a button can wait until after the machine is up and I'm free to test.

Where did you purchase the buttons? It's fine if I have to get them shipped from overseas. The places I have to buy from both cost about 160-180RMB per button, and shipping of about 360RMB, which is rather ridiculous.

**Comment by HelloworldDk** [comment]
我英语水平不佳，直接写英文句子确实有点累，所以我还是换中文吧，应该谷歌翻译能处理）
在中国，我们有个购物网站叫淘宝网，在这个网站上有仿制的按键。
但是那些按键相较官方按键缺少调试，所以手感可能会差一些
一些有能力的玩家自己开模制造了一些按键，出厂手感据说就好很多，相较价格也会比较便宜，他们自己出售也有在一个二手购物平台叫 闲鱼 上面出售，8个按键总价价格大概400RMB，比较便宜，但是使用材料等跟官方和淘宝按键有差异，比如颜色偏向纯白色，特别是里面的弹性金属片有差异，出厂手感可能好一些但是难以后期调整。（他们大多可能都不发货到海外。这些消息都是听说的，我没有实际购买测试过，不知道实际怎么样）

**Comment by HelloworldDk** [comment]
现在我们讨论小组（在腾讯QQ上，内部群，不接外人）也有人复刻出了官方设计的3d打印建模，还没有打印测试但是看起来不错。
也有使用滴胶，硅胶翻模制作的按键壳，效果也不错。
